File: ProyectoFinal/Colores/Python/Colores.py
"""author: @LauraSantacruz"""

a = [11] #PrimeraListaDeColores
b = [11] #SegundaListaDeColores
c = [11] #TerceraListaDeColores
d = [11] #CuartaListaDeColores
        
 #Llenado de listas
        
for i in range(10)
    println("Primera lista de colores ")
    a[i] =int(input("Ingrese color " ,i))        

for i in range(10)
    println("Segunda lista de colores ")
    b[i] =int(input("Ingrese color " ,i)) 

for i in range(10)
    println("Tercera lista de colores ")
    c[i] =int(input("Ingrese color " ,i))

for i in range(10)
    println("Cuarta lista de colores ")
    d[i] =int(input("Ingrese color " ,i))

#Comparacion de listas

for i in range(10)
    if (a[i] == b[i])
        if (b[i] == c[i])
            if (c[i] == d[i])
                println("Son iguales")
            else
                println("Son distintas")
        else 
            println("Son distintas")
    else 
        println("Son distintas")

File: ProyectoFinal/Colores/Java/Colores.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Laura Santacruz
 */
import java.io.*;

public class Colores {

    public static void main(String args[]) throws IOException {
        BufferedReader bufEntrada = new BufferedReader(new InputStreamReader(System.in));
        String a[], b[], c[], d[], e;

        int i;

        a = new String[11];
        b = new String[11];
        c = new String[11];
        d = new String[11];
        for (i = 1; i <= 10; i++) {
            System.out.println("Primera lista de colores ");
            System.out.println("Ingrese color " + i);
            a[i] = bufEntrada.readLine();
        }
        for (i = 1; i <= 10; i++) {
            System.out.println("Segunda lista de colores ");
            System.out.println("Ingrese color " + i);
            b[i] = bufEntrada.readLine();
        }
        for (i = 1; i <= 10; i++) {
            System.out.println("Tercera lista de colores ");
            System.out.println("Ingrese color " + i);
            c[i] = bufEntrada.readLine();
        }
        for (i = 1; i <= 10; i++) {
            System.out.println("Cuarta lista de colores ");
            System.out.println("Ingrese color " + i);
            d[i] = bufEntrada.readLine();
        }
        for (i = 1; i <= 10; i++) {
            if (a[i].equals(b[i])) {
                if (b[i].equals(c[i])) {
                    if (c[i].equals(d[i])) {
                        System.out.println("Son iguales");
                    } else {
                        System.out.println("Son distintas");
                    }
                } else {
                    System.out.println("Son distintas");
                }
            } else {
                System.out.println("Son distintas");
            }
        }

    }
}
